rails/compose.yml

```diff
@@ -1,20 +1,19 @@
-version: '3.8'
 services:
-  backend:
-    container_name: fastapi
+  rails:
     build: ./backend
     env_file:
       - .env.local
     ports:
       - "8080:8080"
     volumes:
-      - ./backend:/code
-  frontend:
-    container_name: nextjs
+      # - ./backend:/code
+      - ./certificates:/code/certificates
+  nextjs:
     build: ./frontend
     env_file:
       - .env.local
     volumes:
-      - ./frontend:/code
+      # - ./frontend:/code
```

Comment:
Should this be commented out?

Reply:
ya thats on purpose. By default we want the containers to run w/o changes from your local env.